graylog2-inputs/src/main/java/org/graylog2/inputs/transports/AmqpTransport.java

```diff
@@ -300,6 +311,16 @@ public ConfigurationRequest getRequestedConfiguration() {
             );
 
             cr.addField(
+                    new NumberField(
+                            CK_HEARTBEAT_TIMEOUT,
+                            "Heartbeat timeout",
+                            ConnectionFactory.DEFAULT_HEARTBEAT,
+                            "Heartbeat timeout interval in seconds (use 0 for no heartbeat)",
```

Comment:
I'd rephrase that to "Heartbeat interval in seconds (use 0 to disable heartbeat)".